report-hazard/index.html

```diff
@@ -0,0 +1,141 @@
+    <link
+      href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
+      integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
+      crossorigin=""
+    />
+    <script
+      src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"
+      integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo="
+      crossorigin=""
+    ></script>
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav>
+      <ul>
+        <li><a title="Select the location" href="#step1">1</a></li>
+
+        <li>
+          <a title="What type of hazard would you like to report?" href="#step2"
+            >2</a
+          >
+        </li>
+
+        <li>
+          <a title="What type of wild animal did you spot?" href="#step3">3</a>
+        </li>
+
+        <li>
+          <a
+            title="Would you like to mention any further details?"
+            href="#step4"
+            >4</a
+          >
+        </li>
+
+        <li>
+          <a
+            title="Please upload photos to help others see the reported hazard visually"
+            href="#step5"
+            >5</a
+          >
+        </li>
+
+        <li>
+          <a title="Review the report before submitting." href="#step6">6</a>
+        </li>
+
+        <li><a href="#step7">7</a></li>
+      </ul>
+    </nav>
+
+    <main>
+      <form id="form">
+        <section class="page" id="step1">
+          <h1>1. Select the location.</h1>
+
+          <div id="map"></div>
+
+          <label>Your current location is:</label>
+          <input type="text" id="name" required />
+
+          <a href="#step2">Continue</a>
+        </section>
+        <section class="page" id="step2">
+          <h1>2. What type of hazard would you like to report?</h1>
+          <input type="radio" name="category" value="Wildlife" />Wildlife
+          <input type="radio" name="category" value="Weather" />Weather
+          <input
+            type="radio"
+            name="category"
+            value="Infrastructure"
+          />Infrastructure
+          <input type="radio" name="category" value="Campsite" />Campsite
+          <a href="#step1">Back</a>
+          <a href="#step3">Continue</a>
+        </section>
+        <section class="page" id="step3">
+          <h1>3. What type of wild animal did you spot?</h1>
+          <input type="radio" name="type" value="Bear" />Bear
+          <input type="radio" name="type" value="Coyote" />Coyote
+          <input type="radio" name="type" value="Coguar" />Coguar
+          <input type="radio" name="type" value="Elk" />Elk
+          <input type="radio" name="type" value="Raccoon" />Raccoon
+          <a href="#step2">Back</a>
+          <a href="#step4">Continue</a>
+        </section>
+
```

Comment:
either add a break line after each section or remove the break line for all of them